simmechanics/data/wrist_mk2/simmechanics2urdf_joints_configfile.csv

```diff
@@ -0,0 +1,4 @@
+joint_name,lower_limit,upper_limit,damping
+l_wrist_yaw_joint,-90.0,90.0,1.0
+l_wrist_pitch_joint,-60.0,50.0,1.0
+l_wrist_roll_joint,-30.0,30.0,1.0
```

Comment:
Can we document this limits somewhere? For the rest of iCub 3 we had this problem that we were not sure what was the reference for this limits, so it would be great if we could document them somewhere, to know where to check for getting the correct values.